### PR TITLE
Added Quote With Marker Package.

### PR DESCRIPTION
**Pull request by dennykorsukewitz**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package does not add key bindings. ** 
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is **[Quote With Marker](https://github.com/dennykorsukewitz/Sublime-QuoteWithMarker)** 

quotewithmarker

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by dennykorsukewitz** [comment]
I already used tags: https://github.com/dennykorsukewitz/Sublime-QuoteWithMarker/tags

File: repository/q.json [hunk 1]
```diff
@@ -444,6 +444,19 @@
 				}
 			]
 		},
+		{
+			"name": "Quote With Marker",
+			"author": "Denny Korsukéwitz",
+			"description": "This package quotes the selected area and adds a custom marker to it.",
```
Comment (braver) [inline repository/q.json:450]: It's easier if you just put a description on your GitHub repo. Package control will know to find and use that.

**Comment by braver** [comment]
> QuoteWithMarker is a Sublime Text 2/3 package 

🤔 That's a bit of a weird audience since ST4 has been out for years already. 

> My package does add key bindings. ** But so hard to trigger.

Haha, yeah, that takes some gymnastics. Good reason to not ship with any at all and let users set something easier to do and to remember 😉 

> There are no packages like it in Package Control

Indeed, I don't think I ever saw something like it. Nice 👍🏻

File: repository/q.json (hunk 1 above)
Comment (dennykorsukewitz) [reply repository/q.json]: Ah, thanks for that hint.
Fixed with: https://github.com/wbond/package_control_channel/pull/8846/commits/c8946b2eeebe69494f09f57f191eeeff9c561b84

**Pull request merged**